README.md

```diff
@@ -53,4 +53,4 @@ Or from source:
       --report-bugs-to [bug address]  An email address or URL where you can report bugs in the untranslated strings.
       -c, --add-comments [tag]        place comment blocks starting with TAG and preceding keyword lines in output file (default: "L10n:").
       --keys [keys]                   JSON keys (comma separated) to use when extracting messages.
-      --ignore-keys [keys]            JSON keys to exclude when extracting messages.
+      --ignore-keys [keys]            JSON keys to exclude when extracting messages.
```

Comment:
I don't think this commit changed the line you meant to change.

Reply:
I couldn't see difference between master and my branch except blank line at the end.

Reply:
The difference was on the line I commented on. This line is a line you added, so I don't know why you'd want to have trailing whitespace on it.